src/css/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-bottom: 2px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 18px;
+  color: #616070;
+  margin-bottom: 14px;
+}
+
+.card__block-reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  padding-top: 2px;
+}
+
+.card__block-price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__sum {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+  font-style: normal;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  border: transparent;
+  height: 40px;
+  color: #ffff;
+  text-transform: uppercase;
+}
+
+.card__button:hover {
+  background: #ffff;
+  border: 1px solid #00acdc;
```

Comment:
Accordingly, the check letter : 9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
You should add .card__button:  border: 1px solid transparent, and  in    .card__button:hover delete. Add to .card__button:hover border-color